### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by deannolan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -431,4 +431,4 @@ links to the four raw transactions published in the example:
 | B's Decred redemption | [49245425967b7e39c1eb27d261c7fe972675cccacff19ae9cc21f434ccddd986](https://mainnet.decred.org/tx/49245425967b7e39c1eb27d261c7fe972675cccacff19ae9cc21f434ccddd986) |
 ## License
 
-These tools are licensed under the [copyfree](http://copyfree.org) ISC License.
\ No newline at end of file
+These tools are licensed under the [copyfree](http://copyfree.org) ISC License.
```
Comment (jrick) [inline README.md]: what changed here?

Comment (BetKingIO) [reply README.md]: I didn't change anything on that line. I just used the github web interface to change the typo and then submitted a PR.

Comment (deannolan) [reply README.md]: Not sure.  I didn't change that line. I only changed the typo using githubs web interface while I was reading it.

Comment (dajohi) [reply README.md]: `git diff` shows
```
-These tools are licensed under the [copyfree](http://copyfree.org) ISC License.
\ No newline at end of file
+These tools are licensed under the [copyfree](http://copyfree.org) ISC License.
```

**Pull request merged**